src/game/GamePlay.as

```diff
@@ -380,7 +380,7 @@ package game
 
             if (MultiplayerSingleton.getInstance().connection.connected && !MultiplayerSingleton.getInstance().isInRoom())
             {
-                MultiplayerSingleton.getInstance().connection.disconnect();
+                MultiplayerSingleton.getInstance().connection.disconnect(true);
```

Comment:
```suggestion
                bool isInSoloMode = true;
                MultiplayerSingleton.getInstance().connection.disconnect(isInSoloMode);
```

Reply:
Using magic bools or magic numbers should be avoided, instead make this readable for what it does right where it is used.